app/controllers/forumthreads_controller.rb

```diff
@@ -80,6 +85,20 @@ def destroy
     redirect_to @thread.forum
   end
 
+  def search
+  end
+
+  def search_redirect
```

Comment:
When I use advanced search and don't change any fields, this is the resulting URL: `https://redstoner.com/forums/threads?author=&content=&reply=&title=`. I don't know about you, but that seems ugly and unnecessary to me.